packages/powersync_flutter_libs/android/build.gradle

```diff
@@ -0,0 +1,69 @@
+// The Android Gradle Plugin builds the native code with the Android NDK.
+
+group 'co.powersync.powersync_flutter_libs'
```

Comment:
Can we make this `com.powersync` instead? Also want to migrate `co.powersync:powersync-sqlite-core` to `com.` at some point.

Reply:
Updated to `com.`